psl-clojure/template/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>psl-clojure</groupId>
+  <artifactId>lein-template</artifactId>
+  <packaging>jar</packaging>
+  <version>2.1.0-SNAPSHOT</version>
+  <name>lein-template</name>
+  <description>FIXME: write description</description>
+  <url>http://example.com/FIXME</url>
+  <licenses>
+    <license>
+      <name>Eclipse Public License</name>
```

Comment:
PSL is under Apache.
See any PSL Java file for details:
https://github.com/eriq-augustine/psl/blob/develop/psl-core/src/main/java/org/linqs/psl/util/Parallel.java#L6